jupyterlab_search_replace/tests/test_handlers.py

```diff
@@ -1,5 +1,6 @@
-import json
+import json, pytest
```

Comment:
Minor styling thingy

```suggestion
import json
import pytest
```